# `isb-cgc-bq.HTAN_versioned.bulkRNAseq_level1_metadata_r4.md`

## Column details

* [STRING]    `Per_Sequence_GC_Content`
  - State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.
* [STRING]    `Encoding`
  - Version of ASCII encoding of quality values found in the file. String
* [STRING]    `HTAN_Data_File_ID`
  - Self-identifier for this data file - HTAN ID of this file HTAN ID SOP (eg HTANx_yyy_zzz)
* [STRING]    `Per_Base_Sequence_Content`
  - State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.
* [STRING]    `Id`
  - Unique identifier of entity e.g. a96059eb-81af-4b24-ae50-9242c0d8f819
* [STRING]    `Basic_Statistics`
  - State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.
* [STRING]    `Adapter_Content`
  - State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.
* [STRING]    `QC_Workflow_Version`
  - Major version for a workflow. String
* [STRING]    `Adapter_Sequence`
  - Base sequence of the sequencing adapter. String
* [STRING]    `Transcript_Integrity_Number`
  - Used to describe the quality of the starting material, esp. in regards to FFPE samples. Number
* [STRING]    `Per_Sequence_Quality_Score`
  - State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.
* [STRING]    `Per_Base_N_Content`
  - State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.
* [STRING]    `Library_Selection_Method`
  - How RNA molecules are isolated.
* [STRING]    `entityId`
  - Synapse ID of entity
* [STRING]    `Adapter_Name`
  - Name of the sequencing adapter. String
* [STRING]    `Per_Tile_Sequence_Quality`
  - State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.
* [STRING]    `Spike_In_Concentration`
  - The final concentration or dilution (for commercial sets) of the spike in mix [PMID:21816910]
* [STRING]    `QC_Workflow_Link`
  - Link to workflow used. String
* [STRING]    `Flow_Cell_Barcode`
  - Flow cell barcode. Wrong or missing information may affect analysis results. String
* [STRING]    `Fragment_Maximum_Length`
  - Maximum length of the sequenced fragments (e.g., as predicted by Agilent Bioanalyzer). Integer
* [STRING]    `Component`
  - Category of metadata (e.g. Diagnosis, Biospecimen, scRNA-seq Level 1, etc.); provide the same one for all items/rows.
* [STRING]    `ROI_Tag`
  - The tag or grouping used to identify the ROI in micro-regional RNA sequencing (if applicable). Must match the ROI tag within the count matrix in level 3.
* [STRING]    `Library_Preparation_Kit_Vendor`
  - Vendor of Library Preparation Kit. String
* [STRING]    `Size_Selection_Range`
  - Range of size selection. String
* [STRING]    `To_Trim_Adapter_Sequence`
  - Does the user suggest adapter trimming?
* [STRING]    `Library_Preparation_Days_from_Index`
  - Number of days between sample for assay was received in lab and the libraries were prepared for sequencing [number]. If not applicable please enter 'Not Applicable'
* [STRING]    `Sequence_Duplication_Levels`
  - State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.
* [STRING]    `Fragment_Standard_Deviation_Length`
  - Standard deviation of the sequenced fragments length (e.g., as predicted by Agilent Bioanalyzer). Number
* [STRING]    `Nucleic_Acid_Source`
  - The source of the input nucleic molecule
* [STRING]    `Library_Strand`
  - Library stranded-ness.
* [STRING]    `Multiplex_Barcode`
  - The barcode/index sequence used. Wrong or missing information may affect analysis results. String
* [STRING]    `Per_Base_Sequence_Quality`
  - State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.
* [STRING]    `Overrepresented_Sequences`
  - State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.
* [STRING]    `Sequence_Length_Distribution`
  - State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.
* [STRING]    `Kmer_Content`
  - State classification given by FASTQC for the metric. Metric specific details about the states are available on their website.
* [STRING]    `HTAN_Parent_Biospecimen_ID`
  - HTAN Biospecimen Identifier (eg HTANx_yyy_zzz) indicating the biospecimen(s) from which these files were derived; multiple parent biospecimen should be comma-separated
* [STRING]    `QC_Workflow_Type`
  - Generic name for the workflow used to analyze a data set. String
* [STRING]    `File_Format`
  - Format of a file (e.g. txt, csv, fastq, bam, etc.)
* [STRING]    `Sequencing_Batch_ID`
  - Links samples to a specific local sequencer run. Can be string or 'null'
* [STRING]    `Filename`
  - Name of a file
* [STRING]    `Library_Layout`
  - Sequencing read type
* [STRING]    `Library_Preparation_Kit_Version`
  - Version of Library Preparation Kit. String
* [STRING]    `Base_Caller_Version`
  - Version of the base caller. String
* [STRING]    `Lane_Number`
  - The basic machine unit for sequencing. For Illumina machines, this reflects the physical lane number. Wrong or missing information may affect analysis results. Integer
* [STRING]    `Micro_region_Seq_Platform`
  - The platform used for micro-regional RNA sequencing (if applicable)
* [STRING]    `Percent_GC_Content`
  - The overall %GC of all bases in all sequences. Integer
* [STRING]    `Read_Indicator`
  - Indicate if this is Read 1 (R1) or Read 2 (R2)
* [STRING]    `Base_Caller_Name`
  - Name of the base caller. String
* [STRING]    `Sequencing_Platform`
  - A platform is an object aggregate that is the set of instruments and software needed to perform a process [OBI_0000050]. Specific model of the sequencing instrument.
* [STRING]    `Fragment_Minimum_Length`
  - Minimum length of the sequenced fragments (e.g., as predicted by Agilent Bioanalyzer). Integer
* [STRING]    `Fragment_Mean_Length`
  - Mean length of the sequenced fragments (e.g., as predicted by Agilent Bioanalyzer). Number
* [STRING]    `RIN`
  - A numerical assessment of the integrity of RNA based on the entire electrophoretic trace of the RNA sample including the presence or absence of degradation products. Number
* [STRING]    `Library_Preparation_Kit_Name`
  - Name of Library Preparation Kit. String
* [STRING]    `DV200`
  - Represents the percentage of RNA fragments that are >200 nucleotides in size. Number
* [STRING]    `Read_Length`
  - The length of the sequencing reads. Can be integer, null
* [STRING]    `Target_Depth`
  - The targeted read depth prior to sequencing. Integer
* [STRING]    `Spike_In`
  - A set of known synthetic RNA molecules with known sequence that are added to the cell lysis mix
* [STRING]    `Total_Reads`
  - Total number of reads per sample. Integer
* [STRING]    `HTAN_Center`
  - Name of HTAN Center e.g. HTAN HTAPP
* [STRING]    `Manifest_Id`
  - Synapse ID of manifest containing file annotation
* [INT64]    `Manifest_Version`
  - Latest version of metadata manifest
* [INT64]    `File_Size`
  - Size of file (bucket object) in bytes
* [STRING]    `md5`
  - md5 hash of file
* [STRING]    `Data_Release`
  - Inclusion in HTAN Data Release e.g. Release 1.0
* [STRING]    `CDS_Release`
  - CDS transfer package in which file was included e.g. v22.6.1.seq

